source/scripts/model/Dungeon.js

```diff
@@ -19,11 +20,36 @@ export class Dungeon {
         })
     }
 }
+/**
+ * Class creates a random dungeon.
+ * @todo Position the stairs better.
+ */
+export class RandomDungeon extends Dungeon {
+    constructor(dungeon) {
+        super(dungeon)
+
+        let gen = new Generator()
```

Comment:
I'd prefer to keep them separate (and would double prefer to pass an instance of the generator to Dungeon through the constructor; didn't right now because of trying to get it out the door), but it would depend on how simple Dungeon is going to stay. if Dungeon is where monster spawning code and stuff like that is going to go, they'd probably be better off separate; however, if Dungeon is going to stay basically empty as it is now then combining them would probably make more sense